File: CS/SvgDataAdapterSample/Form1.cs
using DevExpress.XtraMap;
using System;
using System.Windows.Forms;
namespace SvgDataAdapterSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        #region #SvgFileDataAdapter
        private void Form1_Load(object sender, EventArgs e) {
            Uri baseUri = new Uri(System.Reflection.Assembly.GetExecutingAssembly().Location);
            map.Layers.Add(new VectorItemsLayer {
                Data = new SvgFileDataAdapter {
                    FileUri = new Uri(baseUri, "..\\..\\Data\\countries.svg")
                }
            });
        }
        #endregion #SvgFileDataAdapter
    }
}

File: VB/SvgDataAdapterSample/Form1.vb
Imports DevExpress.XtraMap
Imports System
Imports System.Windows.Forms

Namespace SvgDataAdapterSample

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

#Region "#SvgFileDataAdapter"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            Dim baseUri As Uri = New Uri(Reflection.Assembly.GetExecutingAssembly().Location)
            map.Layers.Add(New VectorItemsLayer With {.Data = New SvgFileDataAdapter With {.FileUri = New Uri(baseUri, "..\..\Data\countries.svg")}})
        End Sub
#End Region  ' #SvgFileDataAdapter
    End Class
End Namespace
